Assets/UI/Scripts/GameTypeButton.gd

```diff
@@ -0,0 +1,29 @@
+tool
+extends HBoxContainer
+class_name GameTypeButton
+
+export(String) var type = "" setget set_new_type
+
+onready var checkbox := $CheckBox
+
+func _ready() -> void:
+	checkbox.text = type
+
+func _process(delta) -> void:
```

Comment:
Prefix unused arguments with a `_` to suppress warnings:

    func _process(_delta) -> void:

There are many other places where this should be done.